DiamondLightSource/SynchWeb
# [LIMS-212] Fix: Persist edited visit comments

JIRA ticket: [LIMS-212](https://jira.diamond.ac.uk/browse/LIMS-212)

Changes:

Fix such that the visitor page persists comments when editing the comment field. Have removed visits page from marionette framework and replaced this with vue.

To test:

    Check user journey is the same.

    Check ISPYB is capturing change in visitor comments.
